app/models/form_submission_attempt.rb

```diff
@@ -107,4 +113,13 @@ def time_to_send
       )
     end
   end
+
+  def simple_forms_form_number
+    @simple_forms_form_number ||=
+      if SimpleFormsApi::NotificationEmail::TEMPLATE_IDS.keys.include? form_submission.form_type
+        form_submission.form_type
+      else
+        SimpleFormsApi::V1::UploadsController::FORM_NUMBER_MAP[form_submission.form_type]
```

Comment:
So this is just expected to be `nil` in cases where the utilizing code has a `form_type` unknown to SimpleFormsApi?

Reply:
Oh never mind, I see this code only gets executed in the case where you are sending a SimpleFormsApi email, so irrelevant in my case

Reply:
Bingo. This is meant to be a way to ensure that we're dealing with a Simple Forms Api form (rather than a different team's form). If it's `nil` then it won't send an email.